lib/opensusebasetest.pm

```diff
@@ -896,6 +896,10 @@ sub wait_boot {
                 send_key 'ret';
             }
         }
+    } elsif (is_bootloader_sdboot) {
```

Comment:
Wouldn't it be better to keep grub2 as default, therefore keeping `else` as it was and move this above it?

Reply:
Can do, but that's only necessary if we assume that `is_bootloader_grub2` is broken and would hide a bug in that case.

I could append an `else die "Unknown bootloader";`?

Reply:
> Can do, but that's only necessary if we assume that `is_bootloader_grub2` is broken and would hide a bug in that case.
> 

valid point

> I could append an `else die "Unknown bootloader";`?

That would be nice, thanks

Reply:
Done